src/sdk/uservarmanager.cpp

```diff
@@ -260,19 +199,24 @@ void UserVariableManager::Arrogate()
                      "%s\n\n"
                      "Please define them."), peList.wx_str());
 
-    PlaceWindow(&d);
+    m_ui->DisplayInfoWindow(_("Global Compiler Variables"), msg);
```

Comment:
Why is this call here? Why don't you put it in the implementation of the OpenEditWindow call?